src/main/java/edu/greenblitz/robotName/subsystems/shooter/FlyWheel/FalconFlyWheel/FalconFlyWheel.java

```diff
@@ -5,43 +5,53 @@
 import com.ctre.phoenix6.hardware.TalonFX;
 import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.FlyWheelInputsAutoLogged;
 import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.IFlyWheel;
-import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.NeoFlyWheel.NeoFlyWheelConstants;
 
 
 import static edu.greenblitz.robotName.subsystems.shooter.FlyWheel.FalconFlyWheel.FalconFlyWheelConstants.*;
 
 
 public class FalconFlyWheel implements IFlyWheel {
-    private TalonFX motor;
+    private TalonFX rightMotor;
+    private TalonFX leftMotor;
+
+
+    VelocityVoltage rightMotorVelocityVoltage = new VelocityVoltage(0).withEnableFOC(true),
+            leftMotorVelocityVoltage = new VelocityVoltage(0).withEnableFOC(true);
 
     public FalconFlyWheel() {
-        motor = new TalonFX(NeoFlyWheelConstants.MOTOR_ID);
-        motor.getConfigurator().apply(CURRENT_LIMITS_CONFIGS);
-        motor.getConfigurator().apply(CLOSED_LOOP_RAMPS_CONFIGS);
-        motor.getConfigurator().apply(MOTION_MAGIC_CONFIGS);
-        motor.setNeutralMode(NEUTRAL_MODE_VALUE);
+        rightMotor = new TalonFX(rightMotorConstants.ID);
+        rightMotor.getConfigurator().apply(rightMotorConstants.CURRENT_LIMITS_CONFIGS);
+        rightMotor.getConfigurator().apply(rightMotorConstants.CLOSED_LOOP_RAMPS_CONFIGS);
+        rightMotor.getConfigurator().apply(rightMotorConstants.MOTION_MAGIC_CONFIGS);
+        rightMotor.setNeutralMode(rightMotorConstants.NEUTRAL_MODE_VALUE);
+
+        rightMotor = new TalonFX(leftMotorConstants.ID);
+        rightMotor.getConfigurator().apply(leftMotorConstants.CURRENT_LIMITS_CONFIGS);
+        rightMotor.getConfigurator().apply(leftMotorConstants.CLOSED_LOOP_RAMPS_CONFIGS);
+        rightMotor.getConfigurator().apply(leftMotorConstants.MOTION_MAGIC_CONFIGS);
+        rightMotor.setNeutralMode(leftMotorConstants.NEUTRAL_MODE_VALUE);
     }
 
     @Override
-    public void setPower(double power) {
-        motor.set(power);
+    public void setPower(double rightPower, double leftPower) {
+        rightMotor.set(rightPower);
+        leftMotor.set(leftPower);
     }
```

Comment:
the rightPower is to the left of the leftPower
```suggestion
    public void setPower(double leftPower, double rightPower) {
        rightMotor.set(rightPower);
        leftMotor.set(leftPower);
    }
```